src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -42,7 +42,7 @@ const ProfileCard = () => {
 				if (!responseProfile.ok) {
 					setIsErrorProfile(true);
 				}
-				const jsonDataProfile = await responseProfile.json();
+				const jsonDataProfile = await responseProfile.json<MemberProfile[]>();
```

Comment:
Question, did we not have problems with this some time ago? (could be wrong)

Reply:
This was the bug on my side, thank you for pointing out. I'll remove those changes as they are fixed in `dev`